TestMod/TestMod.cs

```diff
@@ -8,6 +8,7 @@
 using JotunnLib.Managers;
 using JotunnLib.Utils;
 using TestMod.Prefabs;
+using static JotunnLib.Managers.SkillManager;
```

Comment:
Did we need the static import here?

Reply:
Nice eye, I let VS auto import for me haha, let me fix that.